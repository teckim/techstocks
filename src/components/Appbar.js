import React from 'react';
import { NavLink } from 'react-router-dom';
import { CogIcon, ChevronLeftIcon, MicrophoneIcon } from '@heroicons/react/solid';

const Appbar = () => (
  <div className="flex items-center h-14 px-2 bg-pink-600">
    <NavLink className="basis-1/3 flex" to="/">
      <ChevronLeftIcon className="h-5 w-5" />
    </NavLink>
    <div className="basis-1/3 text-center font-light">Tech Stocks</div>
    <div className="basis-1/3 flex">
      <MicrophoneIcon className="h-5 w-5 mr-4 ml-auto" />
      <CogIcon className="h-5 w-5" />
    </div>
  </div>
);

export default Appbar;
