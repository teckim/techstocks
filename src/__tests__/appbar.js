import Appbar from '../components/Appbar';
import renderWithProviders from '../utils/test-utils';

describe('Appbar components', () => {
  it('should render correctly', () => {
    const { asFragment } = renderWithProviders(<Appbar />);
    const firstRender = asFragment();

    expect(firstRender).toMatchSnapshot();
  });
});
