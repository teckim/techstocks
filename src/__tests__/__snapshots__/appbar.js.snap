// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Appbar components should render correctly 1`] = `
<DocumentFragment>
  <div
    class="flex items-center h-14 px-2 bg-pink-600"
  >
    <a
      aria-current="page"
      class="basis-1/3 flex active"
      href="/"
    >
      <svg
        aria-hidden="true"
        class="h-5 w-5"
        fill="currentColor"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M12.707 5.293a1 1 0 010 1.414L9.414 10l3.293 3.293a1 1 0 01-1.414 1.414l-4-4a1 1 0 010-1.414l4-4a1 1 0 011.414 0z"
          fill-rule="evenodd"
        />
      </svg>
    </a>
    <div
      class="basis-1/3 text-center font-light"
    >
      Tech Stocks
    </div>
    <div
      class="basis-1/3 flex"
    >
      <svg
        aria-hidden="true"
        class="h-5 w-5 mr-4 ml-auto"
        fill="currentColor"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M7 4a3 3 0 016 0v4a3 3 0 11-6 0V4zm4 10.93A7.001 7.001 0 0017 8a1 1 0 10-2 0A5 5 0 015 8a1 1 0 00-2 0 7.001 7.001 0 006 6.93V17H6a1 1 0 100 2h8a1 1 0 100-2h-3v-2.07z"
          fill-rule="evenodd"
        />
      </svg>
      <svg
        aria-hidden="true"
        class="h-5 w-5"
        fill="currentColor"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clip-rule="evenodd"
          d="M11.49 3.17c-.38-1.56-2.6-1.56-2.98 0a1.532 1.532 0 01-2.286.948c-1.372-.836-2.942.734-2.106 2.106.54.886.061 2.042-.947 2.287-1.561.379-1.561 2.6 0 2.978a1.532 1.532 0 01.947 2.287c-.836 1.372.734 2.942 2.106 2.106a1.532 1.532 0 012.287.947c.379 1.561 2.6 1.561 2.978 0a1.533 1.533 0 012.287-.947c1.372.836 2.942-.734 2.106-2.106a1.533 1.533 0 01.947-2.287c1.561-.379 1.561-2.6 0-2.978a1.532 1.532 0 01-.947-2.287c.836-1.372-.734-2.942-2.106-2.106a1.532 1.532 0 01-2.287-.947zM10 13a3 3 0 100-6 3 3 0 000 6z"
          fill-rule="evenodd"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
